ios/FluentUI/Calendar/Views/CalendarViewDayMonthCell.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 public extension Colors {
     struct Calendar {
         public struct Day {
-            public static var textPrimary = UIColor(light: Colors.textSecondary, dark: Colors.textPrimary)
+            public static var textPrimary: UIColor = Colors.textPrimary
             public static var textSecondary: UIColor = Colors.textSecondary
```

Comment:
no this is one that should change

Reply:
Sorry, I don't follow. Which one do you mean that should change?

Reply:
Resolved offline. I should be updating the past date color, not the future date color.